core/marker_manager.js

```diff
@@ -52,6 +52,12 @@ Blockly.MarkerManager = function(workspace){
   this.workspace_ = workspace;
 };
 
+/**
+ * The name of the local marker.
+ * @type {string}
+ */
+Blockly.MarkerManager.LOCAL_MARKER = 'local_marker_1';
```

Comment:
Is this a constant? If yes, then please add `@const`, otherwise I'd suggest renaming to not use upper case (unless there's another reason for it, since it's generally used for const).